macros.tex

```diff
@@ -165,9 +165,10 @@
 \newcommand{\spendattack}{\textsc{spend}\-\text{-}\-\textsc{attack}}
 \newcommand{\bindattack}{\textsc{bind}\-\text{-}\-\textsc{attack}}
 
+\newcommand{\txInclusion}{\tau_\textsf{tx}}
```

Comment:
Not sure why a subscript is needed here. The prover provides a \tau, which verifies. That is all.

Reply:
it's just a different name, because I wanted shorten blockConnection and have them both look similar

Reply:
Let's call it \tau?